src/qt/intro.h

```diff
@@ -47,7 +47,7 @@ class Intro : public QDialog
      * @note do NOT call global GetDataDir() before calling this function, this
      * will cause the wrong path to be cached.
      */
-    static bool showIfNeeded(bool& did_show_intro, bool& prune);
+    static bool showIfNeeded(ArgsManager& args, bool& did_show_intro, bool& prune);
```

Comment:
As `arg` is an in-_out_ parameter, maybe place it at the end of the parameter list?

Reply:
every param in this list is an in-out param

Reply:
Indeed.